api/platformpb/platform.proto

```diff
@@ -112,6 +112,19 @@ message OrganizationEntitlement {
   Platform platform = 11;
 }
 
+message GetContactInformationRequest {}
+
+message GetContactInformationResponse {
+  // CustomerSuccess contains the contanct details of the customer success employee assigned to a customer's account.
+  message CustomerSuccess {
+    string name = 1;
+    string email = 2;
+  }
+  CustomerSuccess customer_success = 1;
```

Comment:
I'm a bit confused about the ticket description. It says about the **_list_** of the contacts, but as far as I see the orgd service can only return one contact person. Is it possible in general that ServiceNow has a list of the contact persons instead of just one?

Reply:
resolving this after the discussion on the DS